content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -242,9 +243,7 @@ PVCs indirectly if they can create Pods, even if they do not have
 permission to create PVCs directly. Cluster administrators must be
 aware of this. If this does not fit their security model, they have
 two choices:
-- Explicitly disable the feature through the feature gate, to avoid
-  being surprised when some future Kubernetes version enables it
-  by default.
+- Explicitly disable the feature through the feature gate.
 - Use a [Pod Security
```

Comment:
@pohly Pod Security Policy is tracked to be deprecated in 1.21. I recommend removing the option to use a Pod Security Policy on this page. What are @kubernetes/sig-docs-en-reviews and @kubernetes/sig-storage-pr-reviews opinions in removing recommending a PSP on this page .

Reply:
I know about the deprecation. But it's the only available option for per-pod control of this feature, so IMHO it is worth mentioning.

Reply:
SIG Auth might have an opinion here too.

Reply:
There are definitely relevant third party solutions in this area; we don't need to list them, but we can mention that various kinds of admission-time restriction on Pods can help restrict PVC creation.

Reply:
Is there already a page with PSP alternatives? If so, then we can point to that as additional options.

Reply:
I think this is OK to merge, and it's _also_ good to highlight this to SIG Security / SIG Auth as an area to consider revising for the v1.21 docs.

Reply:
https://github.com/kubernetes/website/pull/26629 is adding PSP to the deprecation guide for 1.21.

Granular control over specific volume types is not planned for the in-tree replacement for PSP, so pointing people who want that towards the admission webhook documentation seems like a more durable recommendation.

Reply:
I added a "(deprecated in Kubernetes 1.21)" remark for the PSP bullet item and added another one for admission webhook:
```
* Use an [admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/)
  which rejects objects like Pods that have a generic ephemeral
  volume.
```